### ISSUE TITLE
Simplification of Code!! and BUG - top and bottom of paddle

### ISSUE THREAD

**Issue by Ashutosh2012** [BrotatoBoi2point0/Pong#1]
Hello @BrotatoBoi2point0 ,

Just for simplification of code can't we just separate all of the classes into separate modules?!

---

For BUG:- 

    def ball_movement(self, player, opponent, pong_sound):
        # Moving the ball
        self.ball.x += self.speed_x
        self.ball.y += self.speed_y

        # Collision detection and appropriate movement also sounds
        # Collision detection of sides
        if self.ball.top <= 0 or self.ball.bottom >= self.H:
            pygame.mixer.Sound.play(pong_sound)
            self.speed_y *= -1
        if self.ball.left <= 0 or self.ball.right >= self.W:
            pygame.mixer.Sound.play(pong_sound)
            self.speed_x *= -1

        # Collision detection of rects
        if self.ball.colliderect(player) or self.ball.colliderect(opponent):
            pygame.mixer.Sound.play(pong_sound)
            self.speed_x *= -1

Please change the variables accordingly

THANKS!!!!

**Comment by BrotatoBoi2point0** [comment]
Yes, I am planning on making it a little more "modular" so it is a little more readable, At the moment I am focusing on more important projects. Thank you for the advice, I will consider including it whenever I get the time to come back to this side-project.
